### ISSUE TITLE
Fix Bloxroute Implementation on the Front-End for All Palomabots and Curvebots

### ISSUE THREAD

**Issue by taariq** [VolumeFi/paloma#1073]
This is to fix: https://github.com/VolumeFi/paloma/issues/982#issuecomment-1854659659

We need to reimpliment our use of bloxroute rpcs:
The https://virginia.rpc.blxrbdn.com/ url is either outdated or we are not authenticating or it is not useable for generating routes. 

please use the new endpoint listed here for ALL bots https://docs.bloxroute.com/introduction/protect-rpcs and please use authentication required for our Professional Account.

**Comment by taariq** [comment]
From @jamie088 

> I don't think the current CORS error is due to bloxrouter. Because we are getting CORS error in all api calls, not just the bloxroute RPC endpoint.
nest service api call, infura call, sentry call, etc
Not sure but I guess this problem is from Vercel?
We are getting this error in palomabot, curvebot, and palomaswap as well.

**Comment by verabehr** [comment]
```Access to fetch at 'https://virginia.rpc.blxrbdn.com/' from origin 'https://www.curvebot.fi' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.```

**Comment by verabehr** [comment]
When bloxroute was implemented for the very first time (on Palomaswap), the BLOXROUTE_API_AUTH_HEADER_KEY was included as an environment variable. https://github.com/palomachain/paloma-swap-ui/commit/5ceef289158c3c10dbcca58a45dfe8fa6ac10e6c
1. I don't see this variable set in vercel for palomaswap though 
2. I don't see this variable set for palomabot or curvebot

**Comment by taariq** [comment]
good catch.

**Comment by rootedbox** [comment]
https://github.com/VolumeFi/curve-leverage-lending-bot/pull/14
https://github.com/VolumeFi/palomabot/pull/291
https://github.com/VolumeFi/palomabot/pull/292
https://github.com/palomachain/paloma-swap-ui/pull/105

**Comment by rootedbox** [comment]
Auth headers are only for their Cloud-API and Gateway-API.. which we are not using.

**Comment by taariq** [comment]
Superseded by https://github.com/VolumeFi/paloma/issues/1098

**Comment by taariq** [comment]
1. it seems like your url is not entirely correct. its supposed to be https://eth-protect.rpc.blxrbdn.com
2. regarding this problem, does it work if you switch the rpc endpoint to a different one like infura or other nodes? if so, then it probably means that the issue is not on the rpc node side.
3. this simple curl request works for me curl 'https://eth-protect.rpc.blxrbdn.com' --header 'Content-Type: application/json' --data '{"jsonrpc":"2.0", "method":"web3_clientVersion", "params": [], "id":1}'. If the problem still persists, can you share a curl request to help us re-create the problem?

**Comment by taariq** [comment]
Fixed with https://github.com/VolumeFi/paloma/issues/1101
